test_unstructured_ingest/src/s3.sh

```diff
@@ -23,8 +23,7 @@ trap cleanup EXIT
 
 "$SCRIPT_DIR"/check-num-files-expected-output.sh 3 $OUTPUT_FOLDER_NAME 20k
 
-RUN_SCRIPT=${RUN_SCRIPT:-./unstructured/ingest/main.py}
-PYTHONPATH=${PYTHONPATH:-.} "$RUN_SCRIPT" \
+PYTHONPATH=. unstructured-ingest \
```

Comment:
nit: le'ts preserve ability to override script name or PYTHONPATH?
i.e.:

```suggestion
RUN_SCRIPT=${RUN_SCRIPT:-unstructured-ingest}
PYTHONPATH=${PYTHONPATH:-.} "$RUN_SCRIPT" \
```